README.md

```diff
@@ -257,7 +258,7 @@ The `oForms` mixin also allows customisation of the base classname:
 
 - `oFormsWrapper` - an area around the form, which can be coloured.
 - `oFormsMessage` - styles for global form messages.
-- `oFormsPrefixSuffix` - adds ability to affix information/buttons to form fields.
+- `oFormsSuffix` - adds ability to affix information to form fields.
```

Comment:
Is there an example of what 'information' is?

Reply:
🤔 The only design condoned suffix currently is a button so I'll swap out 'information' to 'button'... technically though you could put what you like in that space.